garageScript/c0d3-app
# Persist exercise submissions when user submits exercise submission

Changes:
* When you submit an exercise submission on the DOJO exercises page (/exercises/js0), it will send a GraphQL mutation and persist even if you refresh the page.

How to test:
* Create a DOJO module at /admin/lessons/js0/modules
* Create DOJO exercises at /curriculum/js0/mentor
* Go to /exercises/js0
* Notice how the newly-created exercises are marked as "NOT ANSWERED"
* Click "SOLVE EXERCISES" and answer an exercise and submit it
* Click the back arrow to show the exercise list again
* Notice that when you look at the exercise you answered, it is now marked as "ANSWERED"
* If you refresh the page then the exercises you answered are still marked as "ANSWERED"

This pull request is related to https://github.com/garageScript/c0d3-app/issues/2253
